src/runtime/app.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('reset', () => {
     app.assemble()
     app.reset()
     expect(app.settings.current.server.path).toEqual(app.settings.original.server.path)
-    expect(app.settings.current.schema.connections).toEqual(undefined)
+    expect(app.settings.current.schema.connections).toEqual({ default: { nexusFieldName: 'connection' } })
```

Comment:
I am a bit suspicious of this test. I see nothing methodical here. Unit tests could handle reset testing at lower levels.